packages/gatsby/src/utils/webpack-status.ts

```diff
@@ -0,0 +1,16 @@
+import { flush } from "./page-data"
+
+let isPendingStatus = false
+
+export function isPending(): boolean {
+  return isPendingStatus
+}
+
+export function markAsPending(): void {
+  isPendingStatus = true
+}
+
+export function markAsDone(): void {
+  isPendingStatus = false
+  flush()
```

Comment:
You probably need to check 'isFlushPending' here.

Reply:
This is within `flush` where if `isFlushing` is true, it is a noop

Reply:
Yeah, but isFlushPending is not isFlushing

Reply:
isFlushEnqueud.

Reply:
However you spell that kiu word.

Reply:
And done